vocabularies/UI.json

```
@@ -1828,7 +1828,19 @@
                 "@Core.Description": "The data field's value",
                 "@Validation.DerivedTypeConstraint": [
                     "Edm.PrimitiveType",
-                    "Collection(Edm.PrimitiveType)"
+                    "Collection(Edm.Binary)",
+                    "Collection(Edm.Boolean)",
+                    "Collection(Edm.Date)",
+                    "Collection(Edm.DateTimeOffset)",
+                    "Collection(Edm.Decimal)",
```

```suggestion
                    "Collection(Edm.Decimal)",
                    "Collection(Edm.Double)",
```

If we allow `Edm.Single`, we should also allow `Edm.Double`, given that two of our backends support `Edm.Double` and do not support `Edm.Single`.